components/centraldashboard/app/metrics.ts

```diff
@@ -0,0 +1,48 @@
+import express, {Request, Response} from 'express';
+import client from "prom-client";
+
+const app = express();
+const appName = require("./../package.json").name;
+const appVersion = require("./../package.json").version;
+
+
+export const restHttpRequestDuration = new client.Histogram({
+  name: "rest_http_request_duration_seconds",
+  help: "REST API response time in seconds",
+  labelNames: ["method", "path", "status"],
+});
+
+
+export const restHttpRequestTotal = new client.Counter({
+  name: "rest_http_request_total",
+  help: "Total number of HTTP requests for REST API",
+  labelNames: ["method", "status"],
+});
+
+
+export const appInfo = new client.Gauge({
```

Comment:
Do we need to export this?

Reply:
I think it's useful info about application. Similar think what we did in crud-backend. Do you think it should be removed?

Reply:
I meant that we probably don't need to `export const` and probably `const` is enough since those are not used outside of this file. After the update, the same goes for `restHttpRequestTotal` and `restHttpRequestDuration`. WDYT?

Reply:
Ah you are right, sine now they are not imported. :+1: